public/app/features/dashboard-scene/scene/layout-rows/RowsLayoutManager.tsx

```diff
@@ -32,6 +33,10 @@ export class RowsLayoutManager extends SceneObjectBase<RowsLayoutManagerState> i
     },
     id: 'rows-layout',
     createFromLayout: RowsLayoutManager.createFromLayout,
+
+    getSerializer: () => {
+      return layoutSerializerRegistry.get('RowsLayout').serializer;
```

Comment:
Also experimented with just using `new RowsLayoutSerializer()` here not sure what's best